conf/evolutions/default/1.sql

```diff
@@ -484,9 +484,9 @@ CREATE TABLE  IF NOT EXISTS acs.monitor_event
   module_key VARCHAR(32) NOT NULL,
   module_state INTEGER NOT NULL,
   message VARCHAR(2000)  NULL,
-  starttime TIMESTAMP NOT NULL,
-  endtime TIMESTAMP NOT NULL,
-  lastchecked TIMESTAMP NOT NULL,
+  starttime TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
```

Comment:
Why is this? Does it not work without this change when running against a real mysql?

Reply:
yes. model schema is not consistent with the db schema, or regenerate model again.
`
/** Database column starttime SqlType(TIMESTAMP) */
    val starttime: Rep[java.sql.Timestamp] = column[java.sql.Timestamp]("starttime")
`